larva/src/main/java/org/frankframework/larva/SenderThread.java

```diff
@@ -55,7 +62,14 @@ public SenderThread(ISender sender, Message request, PipeLineSession session, bo
 
 	@Override
 	public void run() {
+		if (securityContext == null) { // Ensure there is always a SecurityContext present, though this gets lost when using the IbisJavaSender
+			securityContext = SecurityContextHolder.getContextHolderStrategy().createEmptyContext();
+			securityContext.setAuthentication(new TestingAuthenticationToken("LarvaSender", ""));
```

Comment:
Ik vermoed dat we dit nu heel vaak doen, kunnen we hier een statische context voor aanmaken en hergebruiken?

Reply:
Om er wat mee te doen? Dat we niet heel vaak een nieuwe dummy maken.
Ik weet niet hoe vaak we er echt een aanmaken maar ik vermoed dat dat soms voor iedere test meerdere keren per test kan gebeuren terwijl het iedere keer een identieke instance is.

Daarom dat ik dacht het misschien 1x als global te maken.